content/influxdb/v2.0/influxdb-templates/monitor.md

```diff
@@ -0,0 +1,123 @@
+ - Internet connectivity from the machine running InfluxDB OSS 2.x and Telegraf to InfluxDB Cloud
+ - Sufficient resource availability to install the template. InfluxDB Cloud Free Plan accounts include [resource limits](/influxdb/cloud/account-management/pricing-plans/#resource-limits/influxdb/cloud/account-management/pricing-plans/#resource-limits).
+
+## Install the InfluxDB OSS Monitoring template
+
+The InfluxDB OSS Monitoring template includes a Telegraf configuration and a dashboard, making it easy to set up and monitor OSS.
+
+1. [Log into your InfluxDB Cloud account](https://cloud2.influxdata.com/), go to **Settings > Templates**, and enter the following template URL:
+
+    `https://raw.githubusercontent.com/influxdata/community-templates/master/influxdb2_oss_metrics/influxdb2_oss_metrics.yml`
+
+2. Click **Lookup Template**, and then click **Install Template**. InfluxDB Cloud imports the template, which includes the following resources:
+   - Dashboard `InfluxDB OSS Metrics`
+   - Telegraf configuration `scrape-infludb-oss-telegraf`
+   - Bucket `oss_metrics`
+   - Check `InfluxDB OSS Deadman`
+   - Labels `influxdb2` and `prometheus`
+
+## Set up InfluxDB OSS for monitoring
+
+By default, InfluxDB OSS 2.x has a `/metrics` endpoint available, which exports Prometheus-style system metrics.
+
+1. If you've changed the default settings to disable the `/metrics` endpoint, re-enable these settings.
+2. To view InfluxDB OSS system metrics in your browser, go to **http://hostname:port/metrics** (by default, the hostname and port is **http://localhost:8086/metrics** if you're on the machine where InfluxDB is hosted). Alternatively, use `curl`to fetch metrics:
+      ```sh
+      curl http://localhost:8086/metrics
+      # HELP boltdb_reads_total Total number of boltdb reads
+      # TYPE boltdb_reads_total counter
+      boltdb_reads_total 41
+      # HELP boltdb_writes_total Total number of boltdb writes
+      # TYPE boltdb_writes_total counter
+      boltdb_writes_total 28
+      # HELP go_gc_duration_seconds A summary of the pause duration of garbage collection cycles.
+      ...
+      ```
+3. Verify you have access to the `/metrics` endpoint for each monitored InfluxDB OSS instance so [Telegraf](/telegraf/latest/) can collect metrics.
+
+4. Add your InfluxDB Cloud account information (url and organization) to your Telegraf configuration by doing the following:
+   1. [In your InfluxDB Cloud account](https://cloud2.influxdata.com/), go to **Load Data > Telegraf**, and click the **InfluxDB Output Plugin** button.
+   2. Copy the URL, token, organization, and bucket, close the window, and then click the **Scrape InfluxDB OSS Metrics** link.
+   3. Under `outputs.influxdb_v2`, replace `URL`, `token`, `organization`, and `bucket`, with your InfluxDB Cloud account information. These parameters use environment variables except for `bucket`.
```

Comment:
How to set environment variables to Cloud info with CLI?

Reply:
Environment variables are set in the OS environment, not with the `influx` CLI. There are multiple ways to set environment variables, but if you were to use just straight bash, it'd be:

```
export INFLUX_TOKEN=<YourAuthToken>
```